test/make-images-push-to-local-registry.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
When will we cleanup these images? I guess this might not be too big of an issue since we are running one build at a time so each build will overwrite the prior images that other builds that ran previously built?

Reply:
I added cleanup commands to new version of cleanup script in node provisioning repo.